notes/index.html

```diff
@@ -37,47 +37,47 @@ <h3>"Fewer and fewer people say they trust each other" | "De moins en moins de g
 
 <h3>"You won't know in advance when the last round is" | "Vous ne saurez pas &agrave; l'avance quand est le prochain tour"</h3>
 
-<p>Dans le jeu de la confiance r&eacute;p&eacute;t&eacute; (aussi connu sous le nom de <em>Dilemme du prisonnier r&eacute;p&eacute;t&eacute;</em>), il est important qu'aucun des joueurs ne sache <em>quand</em> est le dernier tour. Pourquoi ? Pensez-y - au dernier tour, les deux joueurs sauraient que leurs actions n'ont pas de cons&eacute;quence, ils tricheraient donc tous les deux. Mais cela signifie que dans l'avant-dernier tour, leurs actions ne peuvent changer le tour suivant, leurs actions ne changeraient pas le tour suivant, alors ils tricheraient aussi. Maiscela signife que dans l'avant-avant-dernier tour...</p>
+<p>Dans le jeu de la confiance r&eacute;p&eacute;t&eacute; (aussi connu sous le nom de <em>Dilemme du prisonnier r&eacute;p&eacute;t&eacute;</em>), il est important qu'aucun des joueurs ne sache <em>quand</em> est le dernier tour. Pourquoi ? Pensez-y - au dernier tour, les deux joueurs sauraient que leurs actions n'ont pas de cons&eacute;quence, ils tricheraient donc tous les deux. Mais cela signifie que dans l'avant-dernier tour, leurs actions ne peuvent changer le tour suivant, leurs actions ne changeraient pas le tour suivant, alors ils tricheraient aussi. Mais cela signife que dans l'avant-avant-dernier tour...</p>
 
 <h3>"Copycat"</h3>
 
-<p>Cette strat&eacute;gie est mieux connue dans la th&eacute;orie des jeux sos le nom de <em>Tit For Tat</em>. Elle a &eacute;t&eacute; cr&eacute;&eacute;e par Anatol Rapoport en 1980, pour le tournoi de th&eacute;orie des jeux de Robert Axelrod. J'ai d&eacute;cid&eacute; de ne pas utiliser le nom "Tit For Tat" parce que 1) il <em>sonne</em> m&eacute;chant, bien que ce soit une strat&eacute;gie bonne &amp; juste, et 2) une grande partie du public a d&eacute;j&agrave; entendu parl&eacute; de Tit For Tat, donc si ce nom avait &eacute;t&eacute; utilis&eacute;, les joueurs auraient pu parier sur ce personnage parceque "Tit For Tat" leur rappelle quelque chose.</p>
+<p>Cette strat&eacute;gie est mieux connue dans la th&eacute;orie des jeux sous le nom de <em>Tit For Tat</em> ("Œil pour œil"). Elle a &eacute;t&eacute; cr&eacute;&eacute;e par Anatol Rapoport en 1980, pour le tournoi de th&eacute;orie des jeux de Robert Axelrod. J'ai d&eacute;cid&eacute; de ne pas utiliser le nom "Tit For Tat" parce que 1) il <em>sonne</em> m&eacute;chant, bien que ce soit une strat&eacute;gie bonne &amp; juste, et 2) une grande partie du public a d&eacute;j&agrave; entendu parl&eacute; de Tit For Tat, donc si ce nom avait &eacute;t&eacute; utilis&eacute;, les joueurs auraient pu parier sur ce personnage parce que "Tit For Tat" leur rappelle quelque chose.</p>
```

Comment:
Just like with accentuated characters, could you replace `Œ` with `&#338;` and `œ` `&#339;` ? Or you would allow me to make the change myself ?